pkg/bootstrapper/bindata.go

```diff
@@ -0,0 +1,244 @@
+// Code generated for package bootstrapper by go-bindata DO NOT EDIT. (@generated)
```

Comment:
This file is adding a lot of exposed functions to the bootstrapper package, which i feel should not be exposed. Its also adding additional practices of code generation.

Why use this `go-bindata` approach instead of forming a KubeletConfiguration object? https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/config/types.go#L74. I feel like if you take that approach the code will be a lot simpler, and we wont have to deal with the extra hack script, generated code, the template file, etc.

Reply:
Yes. I did look into `KubeletConfiguration` object. As we are not using the ignition file anymore, we would have to manually define most of the fields in the struct. So I am not sure, if that would be a cleaner approach.

We can also opt to not expose the `bindata.go` file and just make its generation as part of the build. So every time we build wmcb, it can be regenerated. This would reduce the extra files.

Reply:
That sounds reasonable, the downside to that approach is what will happen if we want to just test the generated code. I think it's better to expose the endpoint but the build can always opt to generate

Reply:
`openshift/origin` exposes `bindata.go`  in test and has a hack script to update it, as @ravisantoshgudimetla  suggested. 
https://github.com/openshift/origin/blob/d6084de4c1cfb6dfb00c1d746debbfba628822ad/test/extended/testdata/bindata.go

Reply:
`openshift/origin` needn't follow the best practices around this :-)  Anyhow please articulate what your plan is.

Reply:
**Approach:** Didnt expose the `bindata.go` file. It can be generated as part of `build`, `build-wmcb-unit-test` and `build-wmcb-e2e-test`.

**Reasoning:**  Keeping `bindata` as a part of `build-wmcb-unit-test` lets us test only the generated code. `bindata.go` is no longer exposed, resulting in a cleaner codebase.